addons/CDS/script_component.hpp

```diff
@@ -1,3 +1,18 @@
+#define COMPONENT CDS
```

Comment:
Just so I understand correctly... CDS is still going to be prefixed as 'SSS_CDS' right? CDS is just a dynamic dialog thing I was working on that I use in other projects. I just don't want any conflicts since each mod would have an independent version of CDS.

Reply:
Nevermind, I think I figured it out.